RELICENSE/sigiesec.md

```diff
@@ -0,0 +1,16 @@
+# Permission to Relicense under MPLv2 or any other OSI approved license chosen by the current ZeroMQ BDFL
+
+This is a statement by Jovan Bunjevacki that grants permission to
+relicense its copyrights in the libzmq C++ library (ZeroMQ) under the
+Mozilla Public License v2 (MPLv2) or any other Open Source Initiative
+approved license chosen by the current ZeroMQ BDFL (Benevolent
+Dictator for Life).
+
+A portion of the commits made by the Github handle "bjovke", with
+commit author "Jovan Bunjevacki <bjovan@gmail.com>", are
+copyright of Jovan Bunjevacki.  This document hereby grants the libzmq
+project team to relicense libzmq, including all past, present and
+future contributions of the author listed above.
+
+Jovan Bunjevacki
+2017/04/04
```

Comment:
Could you please fix the data here? Given it's a legal thing it's better to be accurate